config/master.cfg

```diff
@@ -485,6 +485,29 @@ scumm_untestable_platforms.append("ds")
 # i: EXTRA_PARAM="--enable-made --disable-mad"
 # k: EXTRA_PARAM="--enable-cruise --disable-mad"
 
+# Switch
+scumm_root_switch = "/opt/toolchains/switch"
+scumm_env_switch = copy.deepcopy(scumm_env)
+scumm_env_switch["PATH"] = "%s/devkitpro/devkitA64/bin:%s/devkitpro/tools/bin:%s" % (scumm_root_switch, scumm_root_switch, DEFAULT_PATH)
+scumm_env_switch["PKG_CONFIG_PATH"] = "%s/devkitpro/portlibs/switch/lib/pkgconfig" % scumm_root_switch
+scumm_env_switch["DEVKITPRO"] = "%s/devkitpro" % (scumm_root_switch)
+
+p_master = {
+	"configureargs": [
+		"--host=switch"
+	],
+	"env": scumm_env_switch,
+	"disttarget": "scummvm_switch.zip",
+	"package": [ "scummvm_switch.zip" ],
```

Comment:
This will result in a zip within a zip. I would suggest removing the zip step from the makefile (or moving it to a different rule), and then doing something like:
```
	"disttarget": "switch_release",
	"package": [ "switch_release/*" ],
	"archive_format": "zip"
```

Reply:
fixed (force pushed)

Reply:
See also https://github.com/scummvm/scummvm/commit/f7ba39c4d519063d6b0fc2968fcf67c5e9600b95